moped-editor/src/views/projects/projectView/ProjectComponents/index.js

```diff
@@ -120,6 +121,21 @@ export default function MapView({
   const { projectComponents, allRelatedComponents } =
     useProjectComponents(data);
 
+  const { errorMessageDispatch, errorMessageState } = useToolbarErrorMessage();
+
+  const { updateClickedComponentIdInSearchParams } = useComponentLinkParams({
+    setClickedComponent,
+    projectComponents,
+    clickedComponent,
+    errorMessageDispatch,
+  });
+
+  /* Bundle updates that need to be made any time a component UI element is clicked */
+  const makeClickedComponentUpdates = (clickedComponent) => {
```

Comment:
super minor, but i kind of want this to live outside of this hook and be exported. at first glance it looks like it is dependent on the other `updateClickedComponentIdInSearchParams` logic but it is not.

Reply:
i pushed a [lil commit](https://github.com/cityofaustin/atd-moped/pull/1168/commits/f4024906b91988b487b189498e94f59498dd311c) to move this out of the parent function ✅

Reply:
ahh yep - not sure what I was looking at yesterday but thanks for updating! 🙏 🙏